Pipfile

```diff
@@ -4,10 +4,6 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
```

Comment:
The same as the previous PR - these [dev-packages] were removed... was this intentional?

Reply:
Added back in dev packages.